### ISSUE TITLE
[Issue]: HDHomerun Legacy tuner cannot load stream

### ISSUE THREAD

**Issue by habeebtc** [jellyfin/jellyfin#10418]
### HDHomerun Legacy tuner cannot load stream

I am trying to get my HDHomeRun Legacy tuner up and running with JellyFin.

It was able to find it, and find all the channels automatically.  But - when I play a channel, it just spins forever.  In the logs, it appears to be hitting a 30 second timeout.

I tested with the latest stable release: 10.8.11.0

I took a packet trace, and can see the stream starting just fine, the server seems to have missed some notification that the stream has actually started.  What I see is that there's a request sent to HDH to start an RTP stream with the address of the server.  The UDP traffic starts, and just keeps going.

### Jellyfin Version

10.8.z

### Environment

```markdown
- OS:Windows Server 2022
- Linux Kernel:
- Virtualization: none
- Clients: Android, WebOS
- Browser: Brave
- FFmpeg Version:
- Playback Method: Browser
- Hardware Acceleration: none
- GPU Model: builtin
- Plugins:
- Reverse Proxy: none
- Base URL: http://192.168.2.155
- Networking: NIC Teaming, 2x gigabit
- Storage: 2x mirrored 10tb volumes for OS/scratch, 2x 14tb mirrored volumes for media storage
```

### Jellyfin logs

```shell
[2023-10-16 17:40:11.757 -05:00] [INF] [39] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile="Anonymous Profile", Path="http://192.168.2.13", AudioStreamIndex=null, SubtitleStreamIndex=null ) => ( PlayMethod=Transcode, TranscodeReason=ContainerNotSupported, VideoCodecNotSupported, AudioCodecNotSupported ) "media:/videos/23f50ea8-47eb-4700-cd53-009b224ac0ce/master.m3u8?MediaSourceId=native_38e2ec422972cf18ae399822dbeeeca8_bc6b703ade5c83c9c39f37fa83a79f71&VideoCodec=h264&AudioCodec=aac,mp3&AudioStreamIndex=-1&VideoBitrate=139808000&AudioBitrate=192000&api_key=<token>&TranscodingMaxAudioChannels=2&RequireAvc=false&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&TranscodeReasons=ContainerNotSupported,%20VideoCodecNotSupported,%20AudioCodecNotSupported"
[2023-10-16 17:40:11.763 -05:00] [INF] [39] Emby.Server.Implementations.LiveTv.LiveTvManager: Opening channel stream from "Emby", external channel Id: "hdhr_11.1"
[2023-10-16 17:40:11.766 -05:00] [INF] [39] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Streaming Channel "hdhr_11.1"
[2023-10-16 17:40:11.768 -05:00] [INF] [39] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: GetChannelStream: channel id: "hdhr_11.1". stream id: "native_38e2ec422972cf18ae399822dbeeeca8_bc6b703ade5c83c9c39f37fa83a79f71" profile: "native"
[2023-10-16 17:40:11.771 -05:00] [INF] [39] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Opening HDHR UDP Live stream from "192.168.2.13"
[2023-10-16 17:40:41.781 -05:00] [INF] [39] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: HDHR UDP stream cancelled or timed out from "192.168.2.13"
[2023-10-16 17:40:41.783 -05:00] [ERR] [46] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Error opening tuner
System.TimeoutException: The operation has timed out.
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunUdpStream.CopyTo(UdpClient udpClient, String file, TaskCompletionSource`1 openTaskCompletionSource, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunUdpStream.CopyTo(UdpClient udpClient, String file, TaskCompletionSource`1 openTaskCompletionSource, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunUdpStream.StartStreaming(UdpClient udpClient, HdHomerunManager hdHomerunManager, IPAddress remoteAddress, TaskCompletionSource`1 openTaskCompletionSource, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunUdpStream.Open(CancellationToken openCancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
```

### FFmpeg logs

```shell
None were generated, as the stream did not start.
```

### Please attach any browser or client logs here

This URL just emits a 500 error:

http://192.168.2.155:8096/Items/23f50ea847eb4700cd53009b224ac0ce/PlaybackInfo?UserId=59e9ed3993f14366b2f3b61b8818bfd3&StartTimeTicks=0&IsPlayback=true&AutoOpenLiveStream=true&MaxStreamingBitrate=140000000

"Error processing request."

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by habeebtc** [comment]
So after poking at this more - the problem may be with my network configuration (the NIC teaming).

Always useful to post these issues anyways - because it may help someone in unexpected ways.  No knowledge ever goes to waste!

So what I think is happening is that when I bind to the IP address for a TCP socket or UDP listener, under the hood the OS is picking a NIC to bind to.  I've just been unfortunate enough that the flows always go to the other one, which has no listener.

I've switched to Switch-Dependent teaming mode, but no dice there yet either (even though my gateway supports it).  Next I am going to try installing a third NIC, assigning it an IP, and then having JellyFin use that for all its stuff.  My NIC teaming is for SMB throughput anyways.

**Comment by habeebtc** [comment]
Dang.  No dice!  

I removed NIC teaming, and actually disabled one NIC, and bound JellyFin to the single active IP.

Same issue it seems:

[2023-10-16 23:08:32.302 -05:00] [INF] [23] Emby.Server.Implementations.LiveTv.LiveTvManager: Opening channel stream from "Emby", external channel Id: "hdhr_11.1"
[2023-10-16 23:08:32.306 -05:00] [INF] [23] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Streaming Channel "hdhr_11.1"
[2023-10-16 23:08:32.532 -05:00] [INF] [23] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: GetChannelStream: channel id: "hdhr_11.1". stream id: "native_38e2ec422972cf18ae399822dbeeeca8_bc6b703ade5c83c9c39f37fa83a79f71" profile: "native"
[2023-10-16 23:08:32.547 -05:00] [INF] [23] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Opening HDHR UDP Live stream from "192.168.2.13"
[2023-10-16 23:08:32.553 -05:00] [INF] [14] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Listening on "192.168.2.155":49152
[2023-10-16 23:08:32.576 -05:00] [INF] [16] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Starting the UDP capture using temp file: "C:\ProgramData\Jellyfin\Server\cache\transcodes\4df3b3c6d4ad4f4f8b6a6ffd1af9a0a4.ts"
[2023-10-16 23:09:02.654 -05:00] [INF] [8] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: HDHR UDP stream cancelled or timed out from "192.168.2.13"
[2023-10-16 23:09:02.697 -05:00] [ERR] [14] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Error opening tuner
System.TimeoutException: The operation has timed out.

(If you are wondering - yes, I am running JellyFin from Visual Studio, and I added a couple of extra log messages)
